lib/system/memalloc.nim

```diff
@@ -33,6 +33,19 @@ when notJSnotNims:
     ## otherwise. Like any procedure dealing with raw memory this is
     ## **unsafe**.
 
+  proc cmpMem*(a, b: pointer, size: Natural): int {.inline, noSideEffect,
+    tags: [], locks: 0, raises: [].}
+    ## Compares the memory blocks ``a`` and ``b``. ``size`` bytes will
+    ## be compared.
+    ##
+    ## Returns:
```

Comment:
needs a runnableExamples

Reply:
runnableExamples doesn't work, I'm getting line 49: invalid identation

Reply:
@planetis-m that's because you need `=` in proc declaration; followup PR welcome

Reply:
That would refactor the system module, I guess there is a good reason it was written that way, if you need runnableexamples so bad you can try.

Reply:
indeed, see https://github.com/nim-lang/RFCs/issues/309